cmd/argocd/commands/configure.go

```diff
@@ -27,6 +27,10 @@ argocd configure --prompts-enabled=false`,
 		Run: func(_ *cobra.Command, _ []string) {
 			localCfg, err := localconfig.ReadLocalConfig(globalClientOpts.ConfigPath)
 			errors.CheckError(err)
+			if localCfg == nil {
+				fmt.Println("No local configuration found.")
```

Comment:
Should this be a `log.Fatal` or at least a `os.Exit(1)`?

Reply:
Thanks for pointing it out. I agree that the code should use `os.Exit(1)` instead of `return`. As for `log.Fatal`, I think using `fmt` is more consistent with the rest of the CLI codebase. Referencing `applicationset.go` as an example of similar error handling. 

https://github.com/argoproj/argo-cd/blob/678f61b8d3a4ce4443a49983f7deae9c80ad8983/cmd/argocd/commands/applicationset.go#L142-L145